schedule/yast/raid/raid1_opensuse_lvm.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name:           lvm+raid1@64bit
```

Comment:
`schedule/yast/lvm+raid1/lvm+raid1_opensuse.yaml`

Reply:
raid1_opensuse_lvm.yaml  done. the dir is the same though